### ISSUE TITLE
`visual_similar_chars`

### ISSUE THREAD

**Issue by Sharad24** [SforAiDl/decepticonlp#7]
The `np.random.choice` function might be a better way to do the picking of the random method.

Also, from documentation and usual practice perspective, it is not standard to define docstrings in the middle of a function.

```
def visual_similar_chars(word, *arg):

    method_pick = np.random.randint(0, len(arg))
    if arg[method_pick] == "unicode":
        """
        get diacritic  characters
        """
        char_array = np.array(list(word))
        diacritic = np.char.add(char_array, u"\u0301")
        return diacritic
    if arg[method_pick] == "visual":
        """
        get visually similar chars. like @ for a. 0 for O.
        """
        return None
```

Should probably be something like the other functions.

**Comment by Sharad24** [comment]
Also, wrap this line in a if statement.
https://github.com/SforAiDl/decepticonlp/blob/2ce1770eec9633c5b3579cb097af60eda9c2d4a1/decepticonlp/transforms/perturb.py#L98

to avoid printing everytime anything is imported from the file

**Comment by rajaswa** [comment]
@shashank-m  cover these issues in your next commit for visual_similar_chars

**Comment by shashank-m** [comment]
Yeah and also I was thinking of creating a dictionary where each character in the alphabet has a  specific Homoglyph to it.

**Comment by Sharad24** [comment]
Closed via #6 and #8
